awapi/autoware_api_msgs/package.xml

```diff
@@ -17,6 +17,10 @@
   <depend>geometry_msgs</depend>
   <depend>diagnostic_msgs</depend>
   <depend>autoware_planning_msgs</depend>
+  
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_cmake_cppcheck</test_depend>
```

Comment:
Since there is no CPP, maybe this can be left out (or alternatively changed to ament_lint_common)

Reply:
Done in c1fd87b. I needed to make some changes to the package.xml so please take a look at that.